tools/cldr/cldr-to-icu/lib/org/unicode/cldr/cldr-api/0.1-SNAPSHOT/cldr-api-0.1-SNAPSHOT.pom

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
```

Comment:
no, let's set up a real maven file on the CLDR side to publish cldr.jar

Reply:
Well sure, that's ideal, but should it hold up submitting this code?

Reply:
let's submit the code with tools/cldr-to-icu/lib being `.gitignore`'d then

Reply:
That's an interesting idea. I'll have to fix the readme to explain why nothing works "out the box", but it is easy enough to install the 2 necessary JAR files. If it only needed to support Unix, I'd just write a bash script to set it all up.

Reply:
it should just be mvn command lines, I'd think. so even if you weren't on unix you could just run the command lines. thanks, appreciate it..

Reply:
Well, you (may) need to build the cldr-api.jar, so that needs documenting too.

Reply:
as of https://github.com/unicode-org/cldr/commit/7a782455a103c45a5ae5b7656b14691fbd9322e1 the API is included in `cldr.jar`

Reply:
> there's now an "api-jar" target in the Ant file

OK, i see you are referring to https://github.com/unicode-org/cldr/pull/107 — I'm not sure this is really needed right now, and as I predicted it is much more complex with an ant-based built. Just use cldr.jar?

Reply:
As mentioned there, the CLDR is a huge dependency mess and probably includes duplicates of classes I'm using via the ICU library dependencies in Maven. As it stands, the Ant file is broken and needs some kind of a fix though.